docs/i18n/ja/docusaurus-plugin-content-docs/current/verifier.md

```
@@ -623,32 +622,32 @@ VP Tokenを検証します。
+  transactionId: string,
   options: VerifyPresentationOptions
-): Promise<VpTokenPayload>
+): Promise<Record<string, VpTokenPayload[]>>
 ```
 
 **パラメータ**:
 - `id`: Verifierの識別子（[VerifierClientId](#VerifierClientId)）
 - `response`: 検証に利用する情報（[Verifierauthorizationresponse](#Verifierauthorizationresponse)）
-- `options`: [VerifyPresentationOptions](#VerifyPresentationOptions)。`expectedAud` の指定が必須です。
+- `transactionId`: `createAuthzRequest` が返した `transactionId`。認可リクエスト時の DCQL クエリを照合するために使用します。
+- `options`: [VerifyPresentationOptions](#VerifyPresentationOptions)
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)